tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_SUBCONUS_Ind_3km_ics_FV3GFS_lbcs_FV3GFS_suite_WoFS_v0.yaml

```diff
@@ -12,6 +14,15 @@ workflow:
   DATE_LAST_CYCL: '2019061500'
   FCST_LEN_HRS: 6
   PREEXISTING_DIR_METHOD: rename
+rocoto:
+  tasks:
+    taskgroups: '{{ ["parm/wflow/prep.yaml", "parm/wflow/coldstart.yaml", "parm/wflow/post.yaml", "parm/wflow/verify_pre.yaml", "parm/wflow/verify_det.yaml"]|include }}'
+    metatask_run_ensemble:
+      task_run_fcst_mem#mem#:
+        walltime: 01:00:00
+    task_get_obs_ccpa:
+    task_get_obs_mrms:
+    task_get_obs_ndas:
```

Comment:
Please remove the empty `task_get_obs_*` entries.  The changes in Mike's PR #864, which was recently merged, will check to see if data is currently staged before attempting to pull data from HPSS.  These empty tasks are no longer required and were removed in PR #864.

Reply:
Done